README.md

```diff
@@ -48,15 +48,27 @@ You can install it from the [release section](https://github.com/kike-canaries/e
 
 ## Requirements
 
+Please first clone the project with all submodules:
```

Comment:
I think this file was "changed" because the `dev` and `master` branches weren’t sync, should we remove this changes from this commit or live them?